src/main.ts

```diff
@@ -5,12 +5,25 @@ import * as fs from 'fs';
 import { window, commands, workspace, OutputChannel, ExtensionContext, ViewColumn, QuickPickItem } from 'vscode';
 import { runInTerminal } from 'run-in-terminal';
 
+const kill = require('tree-kill');

```

Comment:
Let's use an import and make the kill module typed. 
import { kill } from 'kill-tree';

Add a kill-tree.d.ts to the typings folder:
declare module "kill-tree" {
	export function kill(pid: number, signal?: string, callback?: ()=>any);
}